.idea/copyright/copy.xml

```diff
@@ -1,6 +1,6 @@
 <component name="CopyrightManager">
   <copyright>
-    <option name="notice" value="Copyright (C) 2020 Dremio&#10;&#10;Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);&#10;you may not use this file except in compliance with the License.&#10;You may obtain a copy of the License at&#10;&#10;http://www.apache.org/licenses/LICENSE-2.0&#10;&#10;Unless required by applicable law or agreed to in writing, software&#10;distributed under the License is distributed on an &quot;AS IS&quot; BASIS,&#10;WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.&#10;See the License for the specific language governing permissions and&#10;limitations under the License." />
+    <option name="notice" value="Copyright (C) &amp;#36;today.year Dremio&#10;&#10;Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);&#10;you may not use this file except in compliance with the License.&#10;You may obtain a copy of the License at&#10;&#10;http://www.apache.org/licenses/LICENSE-2.0&#10;&#10;Unless required by applicable law or agreed to in writing, software&#10;distributed under the License is distributed on an &quot;AS IS&quot; BASIS,&#10;WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.&#10;See the License for the specific language governing permissions and&#10;limitations under the License." />
```

Comment:
does this mean once a year we will have a big diff which re-writes all headers or we will have a mixed copywrite year depending on when the source was created

Reply:
Nope. It just inserts the current year, when the license is not present.
A "new year" has no effect. However, if you change the copyright text, it will complain.

Reply:
so we will have a mix of 2020, 2021 etc as the copyright year right? That sounds odd to me. Most repos have `Copyright <year>` or `Copyright <year1>-<year2>` right? And they are always uniform across the repo.